### PR TITLE
Codacy fixes

### PR DESCRIPTION
**Pull request by AndyScherzinger**
Fix some codacy warnings about complexity by extracting some logic into separate methods

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
Ready for review, just some minor code moving to shorten existing method for better readability :)

File: src/main/java/com/owncloud/android/authentication/AuthenticatorActivity.java [hunk 1]
```diff
@@ -1461,18 +1462,20 @@ private void onGetServerInfoFinish(RemoteOperationResult result) {
 
         if (!mServerIsValid) {
             // hide old login
-            mOkButton.setVisibility(View.GONE);
-            mUsernameInputLayout.setVisibility(View.GONE);
-            mPasswordInputLayout.setVisibility(View.GONE);
+            showOldLogin(View.GONE);
```
Comment (tobiasKaminsky) [inline src/main/java/com/owncloud/android/authentication/AuthenticatorActivity.java]: Maybe call the function setOldLoginVisibility? As we have both, show/hide and show…

**Comment by tobiasKaminsky** [comment]
:+1: 
Just one small idea

File: src/main/java/com/owncloud/android/authentication/AuthenticatorActivity.java (hunk 1 above)
Comment (AndyScherzinger) [reply src/main/java/com/owncloud/android/authentication/AuthenticatorActivity.java]: DONE -> https://github.com/nextcloud/android/pull/2669/commits/26c2ee9ab9d85baa4805514fc2e12c69e0371d5d

**Comment by mario** [comment]
👍

**Pull request merged**